hibernate-core/src/test/java/org/hibernate/jpa/test/lock/NativeSQLQueryTimeoutTest.java

```diff
@@ -46,9 +49,10 @@ public void test(){
 
 				fail("Should have thrown lock timeout exception!");
 			} catch (Exception expected) {
+				expected.printStackTrace();
 				assertTrue(
 					ExceptionUtil.rootCause(expected)
-						.getMessage().contains("canceling statement due to user request")
+						.getMessage().contains("csanceling statement due to user request")
```

Comment:
So the message is expected to contain the misspelled `csanceling`, right?

Reply:
This was a typo. Thanks for catching it!